cli/mender-artifact/modify.go

```diff
@@ -112,35 +96,50 @@ func modifyName(name, image string) error {
 		return err
 	}
 
-	return debugfsReplaceFile("/etc/mender/artifact_info",
-		tmpNameFile.Name(), image)
+	err = CopyIntoImage(tmpNameFile.Name(), image, "/etc/mender/artifact_info")
+	if errors.Cause(err) == errFsTypeUnsupported && attributeModified {
```

Comment:
Suggestion: Why not move `art` above up one level in the scope hierarchy, and check for `art != nil` or something here, instead of attribute modified?